tools/vite/generate-root-entry-point.ts

```diff
@@ -0,0 +1,47 @@
+import type { LibraryOptions, PluginOption, ResolvedConfig } from 'vite';
+
+export function generateRootEntryPoint(): PluginOption {
+  let viteConfig: ResolvedConfig;
+  return {
+    name: 'package-json-templating',
+    configResolved(config) {
+      viteConfig = config;
+    },
+    async generateBundle() {
+      if (viteConfig.command !== 'build') {
+        return;
+      }
+      const entry = (viteConfig.build.lib as LibraryOptions).entry as Record<string, string>;
+      const classNameMap: Record<string, string> = {
+        SbbOptgroupElement: 'SbbOptGroupElement',
```

Comment:
why do we need this translation here?

Reply:
Because it is the only file that does not match the casing of the file name
`optgroup.ts` => `SbbOptGroupElement`
Usually I infer the class name from the file name, but this does not work for this specific file